### PR TITLE
Enable qps/burst/maxeventageseconds in event exporter

### PR DESCRIPTION
**Pull request by DavidHe1127**
…/maxEventAgeSeconds

### Description of the change

Enable settings of 3 params.

### Benefits

Allow settings of these values. They are available in https://github.com/resmoio/kubernetes-event-exporter#troubleshoot-events-discarded-warning.

### Applicable issues

- fixes #

### Checklist

- [ ] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [ ] Variables are documented in the values.yaml and added to the `README.md` using [readme-generator-for-helm](https://github.com/bitnami-labs/readme-generator-for-helm)
- [ ] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [ ] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

**Comment by DavidHe1127** [comment]
> Hi @DavidHe1127,
> 
> Could you please provide more details about these values?
> 
> The value `config` in the 'bitnami/kubernetes-event-exporter' chart is rendered as a template, so it is not necessary for them to be present in the default `values.yaml` to be configured.
> 
> The real effect of this change would be setting the default config values `kubeQPS: 60, kubeBurst: 60, maxEventAgeSeconds: 10` for all existing deployments.

Did you mean that I can just set them via config and they would work correctly? Code snippet is written in Pulumi

```
    values: {
      config: {
        logLevel: 'error',
        logFormat: 'json',
        kubeQPS: 60,
        kubeBurst: 60,
        maxEventAgeSeconds: 10,
        route: {
          routes: [
            {
              match: [
                {
                  receiver: 'dump',
                },
              ],
            },
          ],
        },
        receivers: [
          {
            name: 'dump',
            stdout: {},
          },
        ],
      },
    },
```

**Comment by migruiz4** [comment]
Yes, additional parameters can be added under the `config:` value and will be added to your deployment.

I can't provide an example using Pulumi at the moment, but here are some other usage examples:
```
# Using helm CLI
$ helm install test bitnami/kubernetes-event-exporter --set config.kubeQPS=60 --set config.kubeBurst=120 --dry-run
...
---
# Source: kubernetes-event-exporter/templates/configmap.yaml
apiVersion: v1
kind: ConfigMap
metadata:
  name: test-kubernetes-event-exporter
  namespace: "default"
  labels:
    app.kubernetes.io/name: kubernetes-event-exporter
    helm.sh/chart: kubernetes-event-exporter-2.3.1
    app.kubernetes.io/instance: test
    app.kubernetes.io/managed-by: Helm
data:
  config.yaml: |
    kubeBurst: 120
    kubeQPS: 60
    leaderElection: {}
    logFormat: pretty
    logLevel: debug
    receivers:
    - file:
        layout: {}
        path: /dev/stdout
      name: dump
    route:
      routes:
      - match:
        - receiver: dump
```

```
#Providing a values.yaml file
$ cat test.yaml                                                             
config:
  kubeQPS: 60
  kubeBurst: 60
  maxEventAgeSeconds: 10
$helm install test bitnami/kubernetes-event-exporter -f test.yaml --dry-run
...
---
# Source: kubernetes-event-exporter/templates/configmap.yaml
apiVersion: v1
kind: ConfigMap
metadata:
  name: test-kubernetes-event-exporter
  namespace: "default"
  labels:
    app.kubernetes.io/name: kubernetes-event-exporter
    helm.sh/chart: kubernetes-event-exporter-2.3.1
    app.kubernetes.io/instance: test
    app.kubernetes.io/managed-by: Helm
data:
  config.yaml: |
    kubeBurst: 60
    kubeQPS: 60
    leaderElection: {}
    logFormat: pretty
    logLevel: debug
    maxEventAgeSeconds: 10
    receivers:
    - file:
        layout: {}
        path: /dev/stdout
      name: dump
    route:
      routes:
      - match:
        - receiver: dump
```

**Comment by DavidHe1127** [comment]
> Yes, additional parameters can be added under the `config:` value and will be added to your deployment.
> 
> I can't provide an example using Pulumi at the moment, but here are some other usage examples:
> 
> ```
> # Using helm CLI
> $ helm install test bitnami/kubernetes-event-exporter --set config.kubeQPS=60 --set config.kubeBurst=120 --dry-run
> ...
> ---
> # Source: kubernetes-event-exporter/templates/configmap.yaml
> apiVersion: v1
> kind: ConfigMap
> metadata:
>   name: test-kubernetes-event-exporter
>   namespace: "default"
>   labels:
>     app.kubernetes.io/name: kubernetes-event-exporter
>     helm.sh/chart: kubernetes-event-exporter-2.3.1
>     app.kubernetes.io/instance: test
>     app.kubernetes.io/managed-by: Helm
> data:
>   config.yaml: |
>     kubeBurst: 120
>     kubeQPS: 60
>     leaderElection: {}
>     logFormat: pretty
>     logLevel: debug
>     receivers:
>     - file:
>         layout: {}
>         path: /dev/stdout
>       name: dump
>     route:
>       routes:
>       - match:
>         - receiver: dump
> ```
> 
> ```
> #Providing a values.yaml file
> $ cat test.yaml                                                             
> config:
>   kubeQPS: 60
>   kubeBurst: 60
>   maxEventAgeSeconds: 10
> $helm install test bitnami/kubernetes-event-exporter -f test.yaml --dry-run
> ...
> ---
> # Source: kubernetes-event-exporter/templates/configmap.yaml
> apiVersion: v1
> kind: ConfigMap
> metadata:
>   name: test-kubernetes-event-exporter
>   namespace: "default"
>   labels:
>     app.kubernetes.io/name: kubernetes-event-exporter
>     helm.sh/chart: kubernetes-event-exporter-2.3.1
>     app.kubernetes.io/instance: test
>     app.kubernetes.io/managed-by: Helm
> data:
>   config.yaml: |
>     kubeBurst: 60
>     kubeQPS: 60
>     leaderElection: {}
>     logFormat: pretty
>     logLevel: debug
>     maxEventAgeSeconds: 10
>     receivers:
>     - file:
>         layout: {}
>         path: /dev/stdout
>       name: dump
>     route:
>       routes:
>       - match:
>         - receiver: dump
> ```

Awesome! Thanks. I will remove them from values file but I think it's still worth it to add them to the comment section as well as in README. What do you think?

I will also change the version to `2.4.1`.

File: bitnami/kubernetes-event-exporter/values.yaml [hunk 1]
```diff
@@ -90,6 +90,9 @@ image:
 ## https://kubernetes.io/docs/concepts/services-networking/add-entries-to-pod-etc-hosts-with-host-aliases/
 ##
 hostAliases: []
+## @param config.kubeQPS Queries per second limit
+## @param config.kubeBurst Up to x number of queries allowed. Burst to roughly match your events per minute QPS to be 1/5 of the burst
+## @param config.maxEventAgeSeconds maximum age of an event that can be processed
```
Comment (migruiz4) [inline bitnami/kubernetes-event-exporter/values.yaml:95]: The `@param` is a reserved word used by the [readme-generator-for-helm](https://github.com/bitnami-labs/readme-generator-for-helm/) to associate the values with their description/metadata.

These values would be removed so this section would need to be replaced.

I would suggest a more generic approach by referencing the upstream documentation and mentioning that this setting is rendered as a template to improve the user experience:

```suggestion
## Kubernetes event exporter configuration, rendered as a template
## ref: https://github.com/resmoio/kubernetes-event-exporter#configuration
```

**Review by migruiz4 — CHANGES_REQUESTED** [review]
Hi there!

I left some suggestions to help you adapt this PR as a description improvement as you proposed.

**Pull request merged**